lib/src/reactive_base_widget.dart

```diff
@@ -1,21 +1,22 @@
 import 'dart:io';
+
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 
 abstract class ReactiveBaseWidget<T> extends StatefulWidget {
   final Stream<T> stream;
-  final T initialData;
+  final T? initialData;
 
   @mustCallSuper
-  const ReactiveBaseWidget(this.stream, this.initialData, {Key key})
-      : assert(stream != null),
-        super(key: key);
+  const ReactiveBaseWidget(this.stream, this.initialData, {Key? key})
+      : super(key: key);
 
   Widget build(BuildContext context, T data);
-  Widget errorBuild(BuildContext context, Object error) {
+
+  Widget errorBuild(BuildContext context, Object? error) {
```

Comment:
Why is error nullable here? Normally the ErrorBuilder should never be called if there is no error or do I miss something here?